.github/workflows/release.yml

```diff
@@ -0,0 +1,122 @@
+name: Package and release
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  DEBIAN_RELEASE: buster
+  DOCKER_PWD: /root
+  DOCKER_IMAGE: debian:${DEBIAN_RELEASE}
+  GO_VERSION: 1.15.5
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [dragonfly, freebsd, linux, netbsd, openbsd]
+        arch: [amd64, arm, arm64, mips, mips64, mips64le, ppc64, ppc64le, s390x ]
+        exclude:
+          - os: dragonfly
+            arch: arm
+          - os: dragonfly
+            arch: arm64
+          - os: dragonfly
+            arch: mips
+          - os: dragonfly
+            arch: mips64
+          - os: dragonfly
+            arch: mips64le
+          - os: dragonfly
+            arch: ppc64
+          - os: dragonfly
+            arch: ppc64le
+          - os: dragonfly
+            arch: s390x
+          - os: freebsd
+            arch: arm64
+          - os: freebsd
+            arch: mips
+          - os: freebsd
+            arch: mips64
+          - os: freebsd
+            arch: mips64le
+          - os: freebsd
+            arch: ppc64
+          - os: freebsd
+            arch: ppc64le
+          - os: freebsd
+            arch: s390x
+          - os: netbsd
+            arch: arm64
+          - os: netbsd
+            arch: mips
+          - os: netbsd
+            arch: mips64
+          - os: netbsd
+            arch: mips64le
+          - os: netbsd
+            arch: ppc64
+          - os: netbsd
+            arch: ppc64le
+          - os: netbsd
+            arch: s390x
+          - os: openbsd
+            arch: arm64
+          - os: openbsd
+            arch: mips
+          - os: openbsd
+            arch: mips64
+          - os: openbsd
+            arch: mips64le
+          - os: openbsd
+            arch: ppc64
+          - os: openbsd
+            arch: ppc64le
+          - os: openbsd
+            arch: s390x
```

Comment:
It seems the exclude list is larger than an explicit inclusion list would be. Maybe dropping the matrix and instead specifying each os-arch variant by hand would be simpler? (of course there would be some redundancy, but simple is good and maintainable).

Reply:
I tried a few different ways of making this work, and wasn't able to find another solution that actually ran each os/arch combo without laying them out in the matrix section (https://github.com/parnic-sks/ubiquitous-memory/actions). I don't like this solution either, so if you find a better way, please advise! :)